frontend/src/widgets/instance/dialogs/InstanceDetail.vue

```diff
@@ -152,6 +147,12 @@ const loadNetworkModes = async () => {
   }
 };
 
+const openDialog = async () => {
+  open.value = true;
+  initFormDetail();
+  await Promise.all([loadImages(), loadNetworkModes()]);
```

Comment:
This line of code has an issue. If Docker is not installed on the system, it will cause an error message to pop up. I will remove it later.